GetRawYieldsDplusDs.py

```diff
@@ -334,6 +334,12 @@
                         hSigmaToFix.GetBinContent(iPt+1)+hSigmaToFix.GetBinError(iPt+1))
                 else:
                     print('WARNING: impossible to fix sigma! Wrong mult factor set in config file!')
+        else:
+            if hSigmaToFix:
+                massFitter[iPt].SetInitialGaussianMean(
+                    hSigmaToFix.GetBinContent(iPt+1)*fitConfig[cent]['SigmaMultFactor'])
+            else:
+                massFitter[iPt].SetInitialGaussianMean(0.008)
```

Comment:
Probably here you meant `SetInitialGaussianSigma`

Reply:
Thanks! Yes indeed this was a typo, fixed in 78c235e